### PR TITLE
remove duplicate tabular objects created when latexing summary_col

### PR DESCRIPTION
**Pull request by natashawatkins**
Best solution I've thought of to #3701 

- uses a regex to find where duplicate tabular objects are created from summary_col, and replaces with two \midrule
- doesn't appear to affect regular regression output as aligns are different

### CODE REVIEW

**Comment by natashawatkins** [comment]
My output now from `summary_col.as_latex()`

image

**Comment by josef-pkt** [comment]
Hi, I had only a brief look
style/pep-8: needs line break to stay below 80 characters per line

> doesn't appear to affect regular regression output as aligns are different

That sounds a bit fragile given that different summarys go through the same code. 
Is it possible to add a flag as attribute to `self` and condition on it, `if self.merge_latex_table: ...`?

Also can you make a simple unit test with just a few columns. Given that there doesn't seem to be anything that changes in the table, like dates or floating point noise, it would be easy just to compare the generated string. The regular expression might easily break if there are any formatting changes, and without unit tests it would be slow to be found.   
(Most of the time we only use smoke tests for summary, but in this case a full test to verify the output wouldn't be difficult.)

**Comment by josef-pkt** [comment]
(insert comment before first comment :)

Thanks, the output looks much better after the change.

**Comment by natashawatkins** [comment]
Thanks for your comments @josef-pkt 

I've gone ahead and added the changes you've suggested

The unit tests simply test whether the \end{tabular}\begin{tabular} still exist or not for OLS vs. summary_col output.

I've also deleted brackets causing the caption error (#3700) because it keeps giving me compilation issues and it's a very simple fix.

File: statsmodels/iolib/tests/test_summary2.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import unittest
+from statsmodels.regression.linear_model import OLS, add_constant
+from summary2 import summary_col
```
Comment (josef-pkt) [inline statsmodels/iolib/tests/test_summary2.py]: needs full import path, fails on all travis runs

**Comment by josef-pkt** [comment]
looks good to me, one import error that needs to be fixed (travis test errors)

can you squash this (so we don't have the reverted file copy and redundant changes)

File: statsmodels/iolib/tests/test_summary2.py (hunk 1 above)
Comment (natashawatkins) [inline statsmodels/iolib/tests/test_summary2.py]: oops! done thanks

**Comment by josef-pkt** [comment]
@natashawatkins  I merged this after rebase and some changes to the unit tests in #3963

Thank you for the contribution

**Pull request closed without merging**